doc/widgets/bagofwords.rst

```diff
@@ -24,9 +24,39 @@ A set of words from corpus.
 Description
```

Comment:
Could you rephrase `A set of words from corpus.` in the `Signal` sections? Maybe something like: A corpus with added BoW features?